set.mm

```diff
@@ -39982,22 +40008,23 @@ among classes ( ~ eq0 , as opposed to the weaker uniqueness among sets,
       UGUH $.
   $}
 
-  $( The difference between a class and itself is the empty set.  Proposition
-     5.15 of [TakeutiZaring] p. 20.  Also Theorem 32 of [Suppes] p. 28.
-     (Contributed by NM, 22-Apr-2004.) $)
-  difid $p |- ( A \ A ) = (/) $=
-    ( wss cdif c0 wceq ssid ssdif0 mpbi ) AABAACDEAFAAGH $.
-
   ${
-    $d x A $.
-    $( Alternate proof of ~ difid .  (Contributed by David Abernethy,
-       17-Jun-2012.)  (Proof modification is discouraged.)
-       (New usage is discouraged.) $)
-    difidALT $p |- ( A \ A ) = (/) $=
+    $d A x $.
+    $( The difference between a class and itself is the empty set.  Proposition
+       5.15 of [TakeutiZaring] p. 20.  Also Theorem 32 of [Suppes] p. 28.
+       (Contributed by David Abernethy, 17-Jun-2012.) $)
```

Comment:
I think people would prefer `Contributed by NM` and then `Revised by David Abernethy` here, don't remember where something like this was said...

Reply:
You're right. It was said when I discouraged the usage of theorems depending on ax-13. I revised the credits in a new commit.